### PR TITLE
Videogames Global Summit 2023

### PR DESCRIPTION
**Pull request by blueberry1312**
Added media queries to both pages (responsive to desktop).
Added interactions with js on the menu button (mobile).
Fixed media query with a script using matchMedia. 
Fixed linters error.
Added video presentation in README file, using loom to record.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,65 +105,68 @@ <h2>
...
+                        <img src="assets/images/program-icon-2.png" alt="program icon 2">
+                    </div>
+                    <div class="program-title">
+                        Exhibition
+                    </div>
+                    <div class="program-description">                    
+                        Appreciate various creations applying CC license of artists, organized from Art Center
+                        Nabi.                
+                    </div>
                 </div>
```
Comment (binhussen) [inline index.html]: - Kindly, Program List is a section, and the program list are article, so let's update it change the `div` tags to `section` and each program list with an `article` tag to make it semantically correct 👍

File: index.html [hunk 2]
```diff
@@ -11,7 +11,76 @@
...
+                        </li>
+                        <li>
+                            <a href="#" class="link-cc-campaign">
+                                CC Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>            
             <div class="headline">
```
Comment (binhussen) [inline index.html:84]: - Kindly your headline is not part of the `header`, please make it out of the `header` and inside its own `section` to make more semantically correct 👍👍

File: index.html [hunk 3]
```diff
@@ -11,7 +11,76 @@
...
                 <h3>
                     "Hello! Sharing world"
```
Comment (binhussen) [inline index.html]: - Kindly hello sharing is your main heading, please make it in `h1`👍 to make more semantically correct 👍

File: index.html [hunk 4]
```diff
@@ -107,6 +179,25 @@ <h2>
...
+                </h2>
+                <img src="assets/images/guide-bar.svg" alt="guide bar image" class="guide-bar-logo">
+                <div class="partners-logo">                    
+                    <img src="assets/images/mozilla.png" alt="mozilla logo" id="mozilla">
+                    <img src="assets/images/google.png" alt="google logo" id="google">
+                    <img src="assets/images/naver.png" alt="naver logo" id="naver">
+                    <img src="assets/images/kakao.png" alt="kakao logo" id="kakao">
+                    <img src="assets/images/airbnb.png" alt="airbnb logo" id="airbnb">
+                </div>
+            </div>
```
Comment (binhussen) [inline index.html]: - Kindly, those partner logos are lists so make it inside `ul` and each items inside `li` to make more semantically correct 👍👍

File: index.html [hunk 5]
```diff
@@ -36,65 +105,68 @@ <h2>
...
+                <div class="program-list">
+                    <div class="program-icon">
+                        <img src="assets/images/program-icon-5.png" alt="program icon 5">
+                    </div>
+                    <div class="program-title">
+                        Ignite
+                    </div>
+                    <div class="program-description">                    
+                        Get opportunities to network with CC affiliates around the world, also after the summit.                            
+                    </div>
```
Comment (binhussen) [inline index.html]: - Kindly, Program List is a section, and the program list are article, so let's update it change the `div` tags to `section` and each program list with an `article` tag to make it semantically correct 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @blueberry1312 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._
- kindly, your live demo version is not rendering the page as expected, please fix it 👍👍 for end users.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binhussen** [comment]
- Kindly your linter is failing when you update after start reviewing it.

**Comment by blueberry1312** [comment]
Hi @binhussen ! I don't know why is not working the deploy pages, can you help with that please?

**Comment by blueberry1312** [comment]
Thanks @binhussen I fix deploy, I forgot to use .gitignore with node_modules and that fix it, I made the semantics changes too so I will request a new review, Thanks you again!

**Comment by arslanbisharat** [comment]
Hi @blueberry1312,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**